pkgs/os-specific/bsd/freebsd/pkgs/rc.nix

```diff
@@ -0,0 +1,87 @@
+{
+  mkDerivation,
+  patchesRoot,
+  lib,
+  sysctl,
+  bash,
+  rcorder,
+  bin,
+  stat,
+  id,
+  protect,
+  mount,
+}:
+let
+  rcDepsPath = lib.makeBinPath [
+    sysctl
+    bin
+    bash
+    rcorder
+    stat
+    id
+    mount
+    protect
+  ];
+in
+mkDerivation rec {
+  path = "libexec/rc";
+
+  # no idea why but the normal derivation setup refuses to produce output even with the CONF* vars set.
```

Comment:
hmm?

Reply:
I took another dive and figured it out. All better!